docs/serving/services/using-queue-extensions.md

```diff
@@ -0,0 +1,80 @@
+# Using Queue Proxy Extensions
+
+Once your cluster is setup to allow services to use Queue Proxy Extensions, a service can decide which extensions it wish to use and how to configure such extensions. Activating and configuring extensions is described here.
```

Comment:
Please refer to the docs style guide re formatting for things like Services https://github.com/knative/docs/blob/main/contribute-to-docs/style-guide/style-and-formatting.md